plugins/source/postgresql/client/list_tables.go

```diff
@@ -0,0 +1,88 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+// this returns the following table in sorted manner:
+// +----------------+-------------+-------------+------------+---------------+-----------+---------------------+
+// | ordinal_position | table_name | column_name | data_type | is_primary_key| not_null  | pk_constraint_name  |
+// +----------------+-------------+-------------+------------+---------------+-----------+---------------------+
+// |              1 | users       | id          | bigint     | YES           | true 		 | cq_users_pk 	  	   |
+// |              2 | users       | name        | text       | NO            | false 	   | 					           |
+// |              3 | users       | email       | text       | NO            | true 		 | cq_users_pk         |
+// |              1 | posts       | id          | bigint     | YES           | true 		 | cq_posts_pk			   |
+// |              2 | posts       | title       | text       | NO            | false 	   | 					           |
+const selectTables = `
```

Comment:
It seems like this query is still the old one, with the bug related to only returning one PK column for tables with composite PKs.